frontend/src/common/queries/censusDirectory.ts

```diff
@@ -0,0 +1,117 @@
+import { useQuery } from "react-query";
+import { ENTITIES } from "./entities";
+import { apiTemplateToUrl } from "../utils/apiTemplateToUrl";
+
+// assumed tier 2 for now
+export interface Project {
+  census_version: string;
+  experiment_name: string;
+  measurement_name: string;
+  submission_date: string;
+  n_features: number;
+  data_type: string;
+  revised_by?: string;
+  model_link?: string;
+  last_updated?: string;
+  title: string;
+  description: string;
+  contact_name: string;
+  contact_affiliation: string;
+  contact_email: string;
+  DOI?: string;
+  publication_info?: string;
+  publication_link?: string;
+}
+
+export interface ProjectResponse {
+  [accessionID: string]: Project;
+}
+
+async function fetchProjects(): Promise<ProjectResponse> {
+  // const url = API_URL + API.CENSUS_DIRECTORY;
+
+  // const response = await fetch(url, {
+  //   ...DEFAULT_FETCH_OPTIONS,
+  //   ...JSON_BODY_FETCH_OPTIONS,
+  // });
+
+  // const result = await response.json();
+  // if (!response.ok) throw result;
+
+  // return result.data;
+  const data: ProjectResponse = {
+    accession_ID_1: {
+      census_version: "2099-08-11",
+      experiment_name: "homo_sapiens",
+      measurement_name: "RNA",
+      submission_date: "2023-01-11",
+      n_features: 2,
+      data_type: "obs_embedding",
+      revised_by: "accession_ID_2",
+      DOI: "10.3352/jeehp.2013.10.3",
+      title: "brief title",
+      description: "...",
+      contact_name: "...",
+      contact_email: "...",
+      contact_affiliation: "...",
+    },
+    accession_ID_2: {
+      census_version: "2099-10-11",
+      submission_date: "2023-01-11",
+      last_updated: "2023-11-02",
+      n_features: 2,
+      data_type: "obs_embedding",
+      measurement_name: "RNA",
+      experiment_name: "homo_sapiens",
+      model_link: "http://example.com",
+      contact_affiliation: "Turing Institute for Biomedical Machine Learning",
+      contact_name: "Haotian Cu",
+      contact_email: "mailto:name@example.com",
+      DOI: "10.3352/jeehp.2013.10.3",
+      title: "BioAI",
+      description:
+        "Ligula imperdiet eget et enim id morbi. Pretium diam risus placerat felis vulputate adipiscing sed integer. Mauris commodo risus scelerisque tempus mi venenatis egestas. Sed at scelerisque vulputate egestas vulputate condimentum libero tempus convallis. Nulla id eget fringilla ultrices pellentesque nunc faucibus condimentum. Ornare porta eget porttitor cum arcu id ultricies id. Massa interdum orci risus arcu mattis massa. Amet metus nibh enim nam pellentesque sagittis diam id quam.",
+    },
+  };
+
+  Object.entries(data).forEach(
+    async ([id, project]: [
+      keyof ProjectResponse,
+      ProjectResponse[keyof ProjectResponse],
+    ]) => {
+      if (!project.DOI) return;
+
+      // include a mailto: query param to insure reliable service
+      const url = apiTemplateToUrl(
+        "https://api.crossref.org/works/{DOI}?mailto=cellxgene@cziscience.com",
```

Comment:
crossref request will be cached alongside our other queries

Reply:
just wanted to confirm you saw the "polite tips" on crossref, which supposedly will help avoid blocking in some cases.

https://www.crossref.org/documentation/retrieve-metadata/rest-api/tips-for-using-the-crossref-rest-api/

Reply:
I was under the impression that we have a crossref API key that was inherited from Meta. CC: @metakuni 

https://github.com/chanzuckerberg/single-cell-data-portal/blob/bc584a92c7585b511e70dc8822a7836b9a54a0b0/backend/common/providers/crossref_provider.py#L34

Reply:
Correct, we have access to [Metadata Plus](https://www.crossref.org/services/metadata-retrieval/metadata-plus/). The credentials can be found in the team vault in 1Password.

Reply:
Hmm we might want to change some code over in cell guide too @atarashansky

Reply:
Since these requests are from the browser, I don't believe we should be using the API key, correct?

Reply:
Since these requests are from the browser, I don't believe we should be using the API key, correct?

Reply:
I don't know what you're trying to do. Is there a spec that explains the use of crossref in this context?

Reply:
No spec.

DOI's are being provided to the client via manifest, we need to convert these DOI's into the format shown[ in the design:](https://www.figma.com/file/DMoBg0zsvktKrOZ3akAnFc/Census?type=design&node-id=774-21604&mode=dev)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/8716829/1aeb0673-2ed3-4cef-a2cb-8bf5279f64a3)